src/api/app/controllers/application_controller.rb

```diff
@@ -460,14 +460,14 @@ def shutup_rails
   end
 
   def set_nobody
-    User.current = User.find_nobody!
+    User.set_current(User.find_nobody!)
```

Comment:
What's happening here if ```User.find_nobody!``` raises an exception?

Reply:
What happened before if it raised an exception? I just changed the API of User model and shied away from too many logic changes in one PR (because I was told repeatedly that you guys prefer step by step approaches).